base/replutil.jl

```diff
@@ -414,8 +414,14 @@ function showerror_ambiguous(io::IO, meth, f, args)
         i < length(p) && print(io, ", ")
     end
     print(io, ") is ambiguous. Candidates:")
+    sigfix = Any
     for m in meth
         print(io, "\n  ", m)
+        sigfix = typeintersect(m.sig, sigfix)
+    end
+    if isa(unwrap_unionall(sigfix), DataType) && sigfix <: Tuple
+        print(io, "\nPossible fix: ")
```

Comment:
```
WARNING: New definition 
    f(...) at ...
is ambiguous with: 
    f(...) at ...
To fix, define 
    f(...)
before the new definition.
```

The error about the ambiguous signatures should happen before this, right? So just the "To fix, define: " part would be analogous to what this line is showing